### PR TITLE
Bug fix: zepMemory needs to return memorykey:[] and not empty string …

### PR DESCRIPTION
**Pull request by rsharath**
If memory not found, loadMemoryVariables returns [], instead needs to return memorykey:[]

### CODE REVIEW

**Comment by rsharath** [comment]
@nfcampos could you please take a look and see if this can be included in the next release please? Thanks

**Comment by danielchalef** [comment]
@jacoblee93 Would it be possible to get this merged soon? We have users complaining about this bug. Thanks!

File: langchain/src/memory/zep.ts [hunk 1]
```diff
@@ -67,8 +67,7 @@ export class ZepMemory extends BaseChatMemory implements ZepMemoryInput {
     } catch (error) {
       // eslint-disable-next-line no-instanceof/no-instanceof
       if (error instanceof NotFoundError) {
-        if (this.returnMessages) return { [this.memoryKey]: [] };
-        return [];
+        return { [this.memoryKey]: [] };
```
Comment (nfcampos) [inline langchain/src/memory/zep.ts:70]: This doesn't look correct, if returnMessages is false it should instead maybe return `{ [this.memoryKey]: '' }` ?

Comment (rsharath) [reply langchain/src/memory/zep.ts:70]: @nfcampos - we need to return an empty list not an empty string because memory is an array of messages.

Comment (rsharath) [reply langchain/src/memory/zep.ts:70]: please let me know if you see an issue with this.

Comment (nfcampos) [reply langchain/src/memory/zep.ts:70]: No that's only true when returnMessages is true. When it is false memory returns a string (that concatenates the messages). See line 99 where it returns the result of calling getBufferString, ie. a string

Comment (rsharath) [reply langchain/src/memory/zep.ts]: ah! thanks, updated to return this.returnMessages ? { [this.memoryKey]: [] } : { [this.memoryKey]: "" };

**Review by nfcampos — APPROVED** [review]

**Pull request merged**